ts/private/ts_project.bzl

```diff
@@ -9,6 +9,27 @@ load(":ts_lib.bzl", "COMPILER_OPTION_ATTRS", "OUTPUT_ATTRS", "STD_ATTRS", "Valid
 load(":ts_config.bzl", "TsConfigInfo")
 load(":ts_validate_options.bzl", _validate_lib = "lib")
 
+# Forked from js_lib_helpers.js_lib_helpers.gather_files_from_js_providers to not
+# include any sources; only transitive declarations & npm linked packages
```

Comment:
Maybe also indicate that in the name? `_gather_declaractions_from_js_providers` or something like that?

Reply:
👍